flight/Modules/Actuator/actuator.c

```diff
@@ -620,8 +620,7 @@ static float mix_channel(int ct, ActuatorDesiredData *desired,
 
 		if (val > 0) {
 			// Apply curve fitting, mapping the input to the propeller output.
-			val = actuatorSettings.MotorInputOutputCurveFit[ACTUATORSETTINGS_MOTORINPUTOUTPUTCURVEFIT_A] *
-					powf(val, actuatorSettings.MotorInputOutputCurveFit[ACTUATORSETTINGS_MOTORINPUTOUTPUTCURVEFIT_B]);
+			val = powapprox(val, actuatorSettings.MotorInputOutputCurveFit);
```

Comment:
The PR references F1, but this affects all targets. Not sure there's any downside to using ```powapprox()``` here, though.